release-notes/opensearch-security.release-notes-1.3.14.0.md

```diff
@@ -0,0 +1,20 @@
+## 2023-12-08 Version 1.3.14.0
+
+Compatible with OpenSearch 1.3.14
+
+### Bug Fixes
+
+* Prevent OptionalDataException from User data structures ([#3725](https://github.com/opensearch-project/security/pull/3725))
+
+### Enhancement
```

Comment:
nit: Enhancement may not be the best categorization since the 1.3 line only contains dependency updates and critical fixes. The integ test framework is a test enhancement, but that's not a category for release notes.

I'm not sure what the best categories should be, but some could probably fall under Maintenance or Bug Fixes.

Reply:
- Consider weighing in on https://github.com/opensearch-project/opensearch-build/issues/4223.  I'm resolving this thread so we can merge and be ready for the patch release.